tests.py

```diff
@@ -44,38 +44,28 @@ def setUp(self):
         self.translator_mock = TranslatorMock(SUBSCRIPTION_KEY)
 
     def test_translate(self):
-        t = self.translator.translate('world', 'en', 'ru')
-        self.assertEqual('мир', t)
+        t = self.translator.translate('world', 'en', 'es')
+        self.assertEqual('Mundo', t)
 
     def test_translate_exception(self):
         self.assertRaises(ArgumentOutOfRangeException, self.translator.translate, 'world', 'en', 'asdf')
 
     def test_translate_array(self):
-        ts = self.translator.translate_array(['hello', 'world'], 'en', 'ru')
-        translations = [t['TranslatedText'] for t in ts]
-        self.assertEqual(['Привет', 'мир'], translations)
+        ts = self.translator.translate_array(['hello', 'world'], 'en', 'es')
+        translations = [t['translations'][0]['text'] for t in ts]
+        self.assertEqual(['Hola', 'Mundo'], translations)
 
     def test_translate_array2(self):
-        ts = self.translator.translate_array2(['hello', 'world', 'Hello. How are you?'], 'en', 'ru')
-        translations = [t['TranslatedText'] for t in ts]
-        self.assertEqual(['Привет', 'мир', 'Привет. Как ваши дела?'], translations)
-        alignments = [t['Alignment'] for t in ts]
-        self.assertEqual(['0:4-0:5', '0:4-0:2', '0:5-0:6 7:18-8:21'], alignments)
-
-    def test_get_translations(self):
-        t = self.translator.get_translations('world', 'en', 'ru')
-        self.assertIsInstance(t, dict)
-        self.assertIn('Translations', t)
+        ts = self.translator.translate_array2(['The answer lies in machine translation.'], 'en', 'es')
```

Comment:
Why this test contains only one phrase to translate? And it would be also great to translate very simple phrases to prevent tests break in case of Translator API changes the translation.

Reply:
I changed to the some phrase as the docs to have the alignment values.